.github/workflows/advance-super.yml

```diff
@@ -34,22 +34,19 @@ jobs:
           # that caused it to wait, reducing push failures down below.
           ref: ${{ github.ref }}
           # We need a token with permission to push.
-          token: ${{ secrets.ZQ_UPDATE_PAT }}
+          token: ${{ secrets.PAT_TOKEN }}
       - uses: ./.github/actions/setup-app
       - run: sudo apt-get -y install whois
-      - run: yarn workspace zui add zed@brimdata/zed#${{ env.zed_ref }}
+      - run: yarn workspace zui add super@brimdata/super#${{ env.super_ref }}
       - run: yarn lint
       - run: yarn test
       - run: yarn build
       - name: End to end tests
         id: playwright
         run: |
-          # Mock for pcap slice opening to avoid hanging Actions Runner.
-          # See https://github.com/brimdata/zui/pull/2987
-          echo "application/vnd.tcpdump.pcap; /usr/bin/true" >> /home/runner/.mailcap
```

Comment:
Since we're not keeping the pcap-centric functionality alive in the app anymore it seems it should be safe to drop this hack.